src/components/LoansByCategory/FeaturedHeroLoan.vue

```diff
@@ -217,14 +217,8 @@ export default {
 		},
 	},
 	created() {
-		// CASH-684 : Experiment : Featured hero loan fundraising thermometer
-		this.experimentVersion = this.experimentData.featured_hero_loan_fundraising_thermometer;
-
-		if (this.experimentVersion === 'variant-a') {
-			this.$kvTrackEvent('Lending', 'EXP-CASH-684-Mar2019', 'a');
-			this.showFundraisingThermometer = false;
-		} else if (this.experimentVersion === 'variant-b') {
-			this.$kvTrackEvent('Lending', 'EXP-CASH-684-Mar2019', 'b');
+		if (Object.prototype.hasOwnProperty.call(this.experimentData, 'featured_hero_loan_fundraising_thermometer') &&
```

Comment:
did you try to use lodash get on this...? Just wondering what
_get(this.experiementData, 'featured_hero_loan_fundraising_thermometer') returned... so you could potentially just check that it === 'variant-c'...

Reply:
PR updated by changing `showFundraisingThermometer` from a property to computed.